data/throughput_benchmarks/workloadc.yaml

```diff
@@ -20,31 +20,31 @@ benchmarks:
       cloud_spanner_config: regional-us-west1
       cloud_spanner_nodes: 3
 
-      # GCE Provisioning: 5 In-Region VMs per Spanner Node
-      ycsb_client_vms: 15
+      # GCE Provisioning: 30 In-Region VMs per Spanner Node
+      ycsb_client_vms: 30
       machine_type: n1-standard-2
       gce_network_name: default
       zone: us-west1-a
 
-      # Data: 100M 1kb rows (100GB total)
-      ycsb_record_count: 100000000
+      # Data: 500M 1kb rows (500GB total)
+      ycsb_record_count: 500000000
       ycsb_field_count: 1
       ycsb_field_length: 1000
       
-      # Use 25 threads/VM in Load phase
-      ycsb_preload_threads: '25'
+      # Use 32 threads/VM in Load phase
+      ycsb_preload_threads: '32'
```

Comment:
I'm not against this change, just worried that it creates a diff here compared to other workload files, which can confuse the users? I know that users can set it to what they want and here we just want to provide sensible defaults.

Reply:
Agreed, but the intent here was to set these numbers exactly to what was used in our benchmarks. wanted to make sure with a high level of confidence that the numbers are reproducible.
going forward, once we have more numbers on Multi region and other workloads, we will be amending the workloads to reflect whatever configuration was used to get to those numbers